app/src/main/java/com/dilekbaykara/tasky/presentation/events/EventDetailsPage.kt

```
@@ -58,6 +84,25 @@ fun EventDetailsPage(
     var fromDateState by remember { mutableStateOf(LocalDate.of(2022, 7, 21)) }
     var toTimeState by remember { mutableStateOf(LocalTime.of(8, 0)) }
     var toDateState by remember { mutableStateOf(LocalDate.of(2022, 7, 21)) }
+    var selectedPhoto by remember { mutableStateOf<Uri?>(null) }
+    val launcher = rememberLauncherForActivityResult(OpenDocument()) { uri: Uri? ->
+        uri?.let {
+            try {
+                context.contentResolver.takePersistableUriPermission(
+                    it,
+                    Intent.FLAG_GRANT_READ_URI_PERMISSION
+                )
+            } catch (_: SecurityException) {}
```

Minor: A debug statement would be helpful for debugging in future. Atm, this will silently fail